package/base-files/files/lib/functions/ipv4.sh

```diff
@@ -0,0 +1,142 @@
+#
+# rewrite of /bin/ipcalc.sh from awk to shell for use as a library
+# in init.d wrappers, and better portability (since busybox awk
+# lacks -E support at this time).
+#
+
+decimal=0
+
+bitcount() {
+    local c="$1"
+
+    c=$((((c >> 1) & 0x55555555) + (c & 0x55555555)))
```

Comment:
Need quotes to pass shellcheck.

Reply:
Problem is that `shellcheck` doesn't know about busybox shell, which is its own thing.

Reply:
I also don't think that quotes matter inside `$((...))` -- at least not the same way they matter elsewhere when it comes to command-injection, etc.

Reply:
You're right about the quotes not being required.
And in fact, no quotes required for arithmetic expansion by shellcheck - I was wrong!

The busybox-compat-issue is very annoying. But most stretches work fine.